.github/workflows/ci.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       # TODO: Port to ROS2
       # DOWNSTREAM_WORKSPACE: "github:ubi-agni/mtc_demos#master github:TAMS-Group/mtc_pour#master"
       # Silent googletest warnings: https://github.com/ament/ament_cmake/pull/408#issuecomment-1306004975
-      UPSTREAM_WORKSPACE: github:ament/googletest#clalancette/update-to-gtest-1.11
+      # UPSTREAM_WORKSPACE: github:ament/googletest#clalancette/update-to-gtest-1.11
```

Comment:
Why does this need to be commented out?

Reply:
CI was failing because it couldn't find `clalancette/update-to-gtest-1.11` so I commented it out. Not sure what the right fix for it would be

Reply:
setup_upstream_workspace
  googletest git https://github.com/ament/googletest clalancette/update-to-gtest-1.11
  === /home/runner/work/moveit_task_constructor/moveit_task_constructor/.work/upstream_ws/src/googletest (git) ===
  Could not checkout ref 'clalancette/update-to-gtest-1.11': fatal: invalid reference: clalancette/update-to-gtest-1.11
  'vcs import --recursive --force /home/runner/work/moveit_task_constructor/moveit_task_constructor/.work/upstream_ws/src' returned with 1
'setup_upstream_workspace' returned with code '1' after 0 min 2 sec```
```

Reply:
Based on this PR being merged (https://github.com/ament/googletest/pull/21) last week, the branch must have been deleted recently. I think you can just change the branch name to `rolling` and you should be back in business.